applications/project_aurras_mvp_phase_2.md

```diff
@@ -0,0 +1,239 @@
+# W3F Grant Proposal
+
+- **Project Name:** Project Aurras - MVP - Phase 2
+- **Team Name:** HugoByte AI Labs Private Limited
+- **Payment Address:** 1G5Fo5KfLSopTFKPRDUZgeCSU5PNY2kkD5 (BTC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+This is an application for a follow-up grant: https://github.com/w3f/Grants-Program/pull/163
+
+### Overview
+
+Aurras is a middleware that acts as an event processor and a low code workflow orchestration platform. Aurras is being pitched as a next-generation system for enabling decentralized push notification. This middleware solution listens to events from blockchain applications and propagates them to a registered pool of MQTT brokers. The broader architecture consist of parachain from which the middleware listens for the events.
+
+Aurras as a part of Substrate / Kusama / Polkadot / Web 3 Ecosystem
+* A core part of Decentralized Push Notification Architecture
+* Can be used for business-critical and production-grade IoT devices to listen to events from blockchain
+* Can be used with microservices across industry verticals
+* Can be expanded to get integrated with data analytics, knowledge visualization, and machine and deep learning (ML/DL) algorithms
+
+### Project Details
+
+#### Architecture
+![](https://storage.googleapis.com/hugobyte-2.appspot.com/architecture.jpeg)
```

Comment:
The top left graphic represents a substrate-based blockchain, right?
![image](https://user-images.githubusercontent.com/5393704/161025545-b62e5973-0951-4209-b15b-6d6555aae752.png)

It might be useful to add a corresponding label here, so it's clear for everybody what it represents.

Reply:
We have added label to the image